pkg/patterns/declarative/reconciler.go

```diff
@@ -345,6 +356,9 @@ func (r *Reconciler) BuildDeploymentObjectsWithFs(ctx context.Context, name type
 	// If Kustomize option is on, it's assumed that the entire addon manifest is created using Kustomize
 	// Here, the manifest is built using Kustomize and then replaces the Object items with the created manifest
 	if r.IsKustomizeOptionUsed() {
+		// set path to root
+		manifestObjects.Path = "."
```

Comment:
Temporary workaround for making `kustomize build` work when having a multi-level directory structure